### PR TITLE
Feature/initial setup

### PR DESCRIPTION
**Pull request by beseven**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Profiles
+# GP Finder
 
-[![Coverage Status](https://coveralls.io/repos/github/nhsuk/profiles/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/profiles?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/nhsuk/gp-finder/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/gp-finder?branch=master)
 
-A collection of health service provider profiles. Initially containing GP practices and Pharmacies.
+A GP finder application. Will used for booking a GP and to find a new GP to register with.
```
Comment (st3v3nhunt) [inline README.md]: Atm, it is only going to be a finder. The booking thing will be a page within the profiles app.

File: .travis.yml [hunk 2]
```diff
@@ -3,12 +3,21 @@ sudo: required
...
   - docker
 
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+script:
+  - docker-compose -f docker-compose-travis.yml build nearby-services-api-tests
```
Comment (st3v3nhunt) [inline .travis.yml]: Think you want `nearby-services-api-tests` to be `web-app-test` (same for line below)

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "profiles",
+  "name": "gp-finder",
   "version": "0.0.1",
   "description": "Web app to display useful information about GP practices Pharmacies",
```
Comment (st3v3nhunt) [inline package.json]: Update description for GP finder

File: README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
 
 ## Environment variables
```
Comment (st3v3nhunt) [inline README.md]: Add `DOCKER_COMPOSE_VERSION` as it is used in `.travis.yml`
Add `COVERALLS_REPO_TOKEN` as is used in `docker-compose-travis.yml`

**Comment by st3v3nhunt** [comment]
[Deploy a dummy application for GP finder journey](https://trello.com/c/CmL7RZfm/738-deploy-a-dummy-application-for-gp-finder-journey)

File: README.md [hunk 5]
```diff
@@ -18,11 +18,12 @@ the application is being run. This is best practice as described by
...
 | `SPLUNK_HEC_ENDPOINT`            | [HTTP Event Collector endpoint](http://dev.splunk.com/view/event-collector/SP-CAAAE7H) |                          | In `production` |
 | `LOG_LEVEL`                      | [bunyan log level](https://github.com/trentm/node-bunyan#levels)                       | Depends on `NODE_ENV`    |                 |
-
+| `DOCKER_COMPOSE_VERSION`         | the docker-compose file version                                                        | 1.11.1 - supports 2.1    | In `travis`     |
+| `COVERALLS_REPO_TOKEN`           | using the coveralls service                                                            | generated for Github user| In `travis`     |  
 
 ## Application development
 
-Start by cloning the repo and all submodules i.e. `git clone https://github.com/nhsuk/profiles.git && cd profiles/ && git submodule update --init --recursive`
+Start by cloning the repo and all submodules i.e. `git clone https://github.com/nhsuk/gp-finder.git && cd gp-finder/ && git submodule update --init --recursive`
```
Comment (st3v3nhunt) [inline README.md]: Not suggesting this is changed but it just occurred to me that the submodule is only required when it is run on Travis.

Comment (beseven) [inline README.md]: I can make the change in another pr while incorporating some of th routing stuff if that is ok with you

Comment (st3v3nhunt) [inline README.md]: Sounds good 👍

**Comment by st3v3nhunt** [comment]
@neilbmclaughlin I'll leave the final review and merge to yourself

**Pull request merged**